### PR TITLE
add newTab page state

### PR DESCRIPTION
**Pull request by cezaraugusto**
Create the resource needed for new tab store consumption such as top sites grid layout (in progress).

Auditors: @bbondy

### CODE REVIEW

File: docs/state.md [hunk 1]
```diff
@@ -189,6 +189,9 @@ AppStore
       }],
       timestamp: number
     }
+  },
+  about: {
+    newtab: {...}
```
Comment (bbondy) [inline docs/state.md]: Please document the subvalues here too

Comment (bbondy) [inline docs/state.md]: The ones you expect you will have.

**Comment by bbondy** [comment]
++ otherwise

**Comment by cezaraugusto** [comment]
done

**Comment by bbondy** [comment]
Thanks!

**Pull request merged**